dome.py

```diff
@@ -141,9 +141,47 @@
 			print("Beam interference #2")
 		if(GPIO.input(37) == 1):
 			print("solid #2")
+			
+#this is an attempt at introducing an event to indicate rising/falling transition
+#Not Tested 
+# option 1
+# def my_callback(channel):
+#	if GPIO.input(37) == 0:
+#		print("Falling edge")
+#		
+#	if GPIO.input(37) != 0:
+#		print("Rising edge")
+#	notch_ir = GPIO.input(37)
+#	return notch_ir
+# then in clockwise and counterwise we have if notch_ir = 1 notches == notches + 1 
+# but I dont see how this is different from what we have now? this just seems the same? 
+# 
+# GPIO.add_event_detect(37, GPIO.BOTH, callback = my_callback)
+
+# option 2
+# def my_callback(channel):
+#	if GPIO.input(37) == 0:
+#		print("Falling edge")
+#		notch_ir = 0
+#		return notch_ir
+#		
+#	if GPIO.input(37) != 0:
+#		print("Rising edge")
+#		notch_ir = 1
+#		return notch_ir
+#
+# GPIO.add_event_detect(37, GPIO.BOTH, callback = my_callback)
+
+# option 3 
+# GPIO.wait_for_edge(37, GPIO.RISING)
+# notches == notches + 1
```

Comment:
What is notches here? It needs to be defined.

Reply:
notches is what we were using in the go location function like 
go_location(azimuth)	
#	go_clockwise() #goes clockwise until notches = azimuth notch data 
#	notches = 0 
#	while notches < azimuth 
#		if(GPIO.input(37) == 0):
#			notches == notches +1 
#			print(notches)
#		else: 
#			notches = notches 
so we would set it to zero and then count

Reply:
rip idk what happened with the font size there

Reply:
Ah, yeah. This would have to go below where it is defined, or we have to define it earlier.